Stuck in retry at ErrTooManyReplica's again

```
[ERROR] Request: invoke_deal, received 403 Forbidden: {'request_id': 'f18db7b2-6a39-464d-af3b-92312408e44f', 'response_timestamp': '2023-06-01T12:47:32.643004901Z', 'response_state_epoch': 2910490, 'response_code': 403, 'error_code': 4023, 'error_slug': 'ErrTooManyReplicas', 'error_lines': [' Piece baga6ea4seaqpgm64hddhtvuql6ptgmjf2yb7p7qzanh3fin5vz6i3mesmz7bkmi is over-replicated according to all selected replication rules '], 'response': {'tenant_replication_states': [{'tenant_id': 13, 'tenant_client_id': 'f02090659', 'tenant_max_in_flight_bytes': 4398046511104, 'actual_in_flight_bytes': 481036337152, 'tenant_max_total': 6, 'tenant_max_per_org': 2, 'tenant_max_per_metro': 2, 'tenant_max_per_country': 4, 'tenant_max_per_continent': 4, 'actual_total': 6, 'actual_within_org': 0, 'actual_within_metro': 0, 'actual_within_country': 2, 'actual_within_continent': 3, 'sp_holds_qualifying_deal': False}]}}
[INFO] RETRYING: Finished call to '__main__.make_request' after 2260.965(s), this was the 362nd time calling it.
```

Running this version;
https://github.com/jcrowthe/ace_of_spades/blob/022d19e1855f9515fe7bc58e5936d0bc6163e56b/ace_of_spades.py

--- a/ace_of_spades.py
+++ b/ace_of_spades.py
@@ -342,10 +342,10 @@ def make_request(*, url: str, method: str, parameters: dict, log_name: str, mine
             raise Exception("Auth token is in the future.")
         else:
             log_request.error(f"{log_name}, received 401 Unauthorized: {res}")
-            raise None
+            return None
     if response.status_code == 403:
         log_request.error(f"{log_name}, received 403 Forbidden: {res}")
-        raise None
+        return None
 
     return res
 

